website/_layouts/mainrenaissance.html

```diff
@@ -141,7 +141,13 @@
   <br/>
   <br/>
 
-  <!-- summary of latest benchmark results -->
+  <div id="measurements" class="text-snippet">
+    <img src="https://github.com/renaissance-benchmarks/measurements/raw/master/stripe.png"/>
```

Comment:
One detail - with the height 500px, the width will be ~1667px, and our current CSS tiling framework is 960px wide.

If we reduced the width to at least 1100px, the figure would spill only a little, while still being readable.

Reply:
Example (by editing the Github page directly in the browser):

![screenshot-21-54-10](https://user-images.githubusercontent.com/151774/57251205-8652af00-7049-11e9-938d-f703c92e3fd9.png)

Reply:
Tried on a local copy but width 1100px does not look very readable in my browser (see screenshot), is it possible we have different zoom settings ?

![screenshot](https://user-images.githubusercontent.com/6158703/57251676-b3ec2800-704a-11e9-84ef-5350d07787c0.png)

Reply:
If we do not want to exceed the 960px I think we should use two rows.